chapters/en/slides/module5/module5_06.md

```diff
@@ -64,53 +53,88 @@ The fork workflow is used in many open source projects where there is often a sm
 
 <img src='/module5/fork.png' width="80%" alt="404 image" />
 
-<center/>
+</center>
 
-Notes: To start developing on a fork, first clone it **(your fork, not the upstream repo)**. It is good practice to always create a branch when you are about to add changes to a fork, and in many projects this is required, so they will not accept a pull request that comes from your `main` branch. This is because it is easier to keep track of branches with descriptive names compared to having all pull request branches names `main`, but coming from different people.
+Notes:
+To start developing on a fork, first clone it to your computer **(your fork, not the upstream repo)**. It is good practice to always create a branch when you are about to add changes to a fork, and in many projects this is required, so they will not accept a pull request that comes from your `main` branch. This is because it is easier to keep track of branches with descriptive names compared to having all pull request branches names `main`, but coming from different people.
 
 When you are working on a branch you have to push your branch to GitHub and ask for it to be merged via a pull request. Since you don't have write access to the upstream repo, you have to request they the maintainer pull your code (remember that merging is part of what happens during a pull, so this is similar to what we did with branches via git merge, just that the code first needs to be fetched from your forked repo)
 
 ---
 
-## Pushing a branch to GitHub (terminal)
-```
-git push
-```
+## Keeping your remote (origin) up to date with the upstream remote
 
-```out
-fatal: The current branch lecture-todo has no upstream branch.
-To push the current branch and set the remote as upstream, use
 
-    git push --set-upstream origin new-branch
+<center>
 
-```
+<img src='/module5/fetch-upstream.png' width="80%" alt="404 image" />
 
+</center>
 
-Notes: To push a feature branch to GitHub, you can try typing git push, just as when we pushed to the master branch on GitHub It will show you the error message you can see in the slide.
 
-Git is telling you that there is no branch with this name currently on your GitHub repo, so it asks you to be explicit about creating a new branch and which name to give it. One gotcha here is that the word "upstream" is not referring to the GitHub branch you forked from. Instead, it is referring to a branch that will be created on your repo (origin). Technically, this seemingly ambiguous use of the word "upstream" is correct (the repo you forked from is upstream of your remote repo, and the branch on your remote repo is upstream of the branch on your local repo), but it can be confusing that the same name is used so it might help to think of the command above as `git push --set-repo-branch <repo-nickname> <new-branch-name>` instead.
```

Comment:
Hmm, I don't quite understand when this can happen. I don't remember that I have ever needed to run that command and pushing to a forked repo should work just as pushing to any GitHub repo.

Reply:
Ah ok, you can put this somewhere else when you do your additional revisions before/while recording